src/gui/src/layoutViewer.cpp

```diff
@@ -887,19 +887,19 @@ void LayoutViewer::selectAt(odb::Rect region, std::vector<Selected>& selections)
                                    region.yMax(),
                                    instanceSizeLimit());
 
-  if (options_->areITermsVisible() && options_->areITermsSelectable()) {
+  if (options_->areITermsVisible()) {
     for (auto& [box, inst] : insts) {
       if (options_->isInstanceVisible(inst)
           && options_->isInstanceSelectable(inst)) {
-        selections.push_back(gui_->makeSelected(inst));
-      }
-      for (auto iterm : inst->getITerms()) {
-        Rect iterm_bbox = iterm->getBBox();
-        if (region.xMin() > iterm_bbox.xMax()
-            || region.xMax() < iterm_bbox.xMin()
-            || region.yMin() > iterm_bbox.yMax()
-            || region.yMax() < iterm_bbox.yMin()) {
-          selections.push_back(gui_->makeSelected(iterm));
+        for (auto iterm : inst->getITerms()) {
+          Rect iterm_bbox = iterm->getBBox();
+          if (region.xMin() <= iterm_bbox.xMax()
+              && region.xMax() >= iterm_bbox.xMin()
+              && region.yMin() <= iterm_bbox.yMax()
+              && region.yMax() >= iterm_bbox.yMin()) {
+            selections.push_back(gui_->makeSelected(iterm));
+          }
```

Comment:
Use [`Rect::contains()`](https://github.com/The-OpenROAD-Project/OpenROAD/blob/74b75f764fb96332c08f7af9c073b43cba617c4d/src/odb/include/odb/geom.h#L266-L267)

Reply:
I replaced it with Rect::instersects() since it matches the logic I was using. Intersect will allow for a points on the edge of an iterm bbox to count as a selection, as well as rubber bands partially covering an iterm bbox.

Reply:
It depends on which Rect's perspective but either works.